templates/_base.html.twig

```diff
@@ -51,9 +51,14 @@
       {% block footer %}
         <p>
           Kind regards,<br>
-          The Big Give Team<br>
+          Big Give Team<br>
           www.thebiggive.org.uk<br>
         </p>
+
+        {% if donorUsername %}
```

Comment:
What happens with the existing emails here? Do we want all emails to have this reminder? Do we actually benefit from the complexity of checking whether card details are saved, given that all donors now have some PII and card info held by Stripe (whether for reuse or not).

Reply:
@NoelLH I think we _should_ add a condition here. I was about to resubmit the PR with the condition removed, and the sentence included in all emails, but I realised it doesn't make sense to have it included in the `pledger-success` email template. As such, I have kept the inclusion of a condition.
I opted to _not_ make the new param `donorCardIsSaved` as mentioned in your [other comment](https://github.com/thebiggive/mailer/pull/81#discussion_r962541421) because it'd mean that to include this sentence to existing templates, we'd need to include the param in the `requiredParams` (for full refund, tip refund, and donation success templates) _and_ make a corresponding SF ticket to include the parameter when calling out from SF, etc. As such, I've made the condition simply with the `donorFirstName` parameter, which all templates _already_ have, except the `pledger-success` one.